src/lib/viewers/doc/PresentationPreloader.js

```diff
@@ -3,6 +3,9 @@ import { CLASS_INVISIBLE, CLASS_BOX_PREVIEW_PRELOAD_WRAPPER_PRESENTATION } from
 import { setDimensions } from '../../util';
 
 class PresentationPreloader extends DocPreloader {
+    /** @property {HTMLELement} - Maximum auto-zoom scale - set to 0 for no limit */
+    maxZoomScale;
```

Comment:
AFAIK setting to 0 here will override the default, if set in the parent class in the same manner. See above comment. This cleans up the constructor quite a bit.